server/README.md

```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
 
 `kubectl create secret generic <your-secret-name> --from-env-file=<path-to-env-file>`
 
-## 4. Deploy changes to the Kubernetes Cluster
+## 4. Deploy and run the smoke tests on Staging
 
 We use [helm](https://helm.sh/docs/) to manage our deployments. Helm allows us to deploy different instances of the application corresponding to dev, staging, and production environments as different 'releases'. Before updating production, we run smoke tests to verify that there's nothing obviously wrong with a release candidate.
 
-### Deploy and run the smoke tests on Staging
-
 1. Run `ssh brave@sensors-admin.brave.coop`
 
 1. cd into the `~/BraveSensor/server` directory and run `git checkout production && git pull origin production` to get the latest version of the helm chart
 
-1. Run the command `helm upgrade staging --set secretName=sensor-staging --set image.tag=<your new container tag> ~/BraveSensor/server/sensor-helm-chart` to deploy the latest version to staging
+1. Run the command `helm upgrade staging --set secretName=sensor-staging ~/BraveSensor/server/sensor-helm-chart` to deploy the latest version to staging
 
 1. Run the smoke test script as described [below](#running-smoke-tests-on-the-kubernetes-cluster)
 
 1. Verify that the deployment shows the expected behavior before proceeding
 
-### Update production
+## 5. Deploy the firmware changes and run a test on the BetaTest Borons
+
+1. on your local machine, open Visual Studio Code in the `firmware/boron-ins-fsm` directory and open the Particle Workbench extension
+
+1. run `git checkout production && git pull origin production` to get the latest version of the the code
+
+1. click Target --> "Configure for device" and type `2.0.0` to choose the OS and then `boron` to choose the device
+
+1. click Compile --> "Local Compile"
+
+1. copy the generated `/target/2.0.0/boron/boron-ins-fsm.bin` file somewhere to keep for reference. Rename to `v<version number>\_dev.bin`
```

Comment:
Providing an example of the bin file can be renamed to might be useful, for example `v50001_dev.bin`

Reply:
Done.